google-cloud-compute-v1/samples/pagination.rb

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START compute_images_list_page]
+# [START compute_images_list]
+
+require "google/cloud/compute/v1"
+
+# [END compute_images_list]
+# [END compute_images_list_page]
+
+# [START compute_images_list]
+# Prints a list of all non-deprecated image names available in given project.
+#
+# @param [String] project project ID or project number of the Cloud project you want to list images from.
+def print_images_list project:
+  client = ::Google::Cloud::Compute::V1::Images::Rest::Client.new
+
+  # Make the request to list all non-deprecated images in a project.
+  request = {
+    project: project,
+    # max_results indicates the maximum number of items that will be returned per page.
+    max_results: 3,
```

Comment:
Based on arguments provided by @amanda-tarafa in [this comment](https://github.com/GoogleCloudPlatform/dotnet-docs-samples/pull/1445/files#r687921807), please change to 100.